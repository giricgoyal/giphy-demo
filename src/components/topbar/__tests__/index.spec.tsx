import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'

import Navbar from '../index'

describe('<Navbar />', () => {
    let container

    beforeEach(() => {
        ;({ container } = render(<Navbar />))
    })

    test('should match snapshot', () => {
        expect(container).toMatchSnapshot()
    })
})
