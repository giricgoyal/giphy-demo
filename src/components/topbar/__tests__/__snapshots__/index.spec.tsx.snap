// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should match snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand navbar-light"
    data-testid="navbar"
  >
    <div
      class="container"
    >
      <a
        class="navbar-brand"
        href="#"
      >
        GIPHY-DEMO
      </a>
    </div>
  </nav>
</div>
`;
